test/docker/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
 
   tigris_search:
     container_name: tigris_search
-    image: typesense/typesense:0.23.0
+    image: typesense/typesense:0.23.1
```

Comment:
Should I just change here @rbarabas?

Reply:
In this repo I think yes.

Reply:
Fix `scripts/install_local_docker_deps.sh`

Reply:
yes, it is part of the diff.